xanitizer.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analyzers
 description=Import Xanitizer security reports
 homepageUrl=https://github.com/RIGS-IT/sonar-xanitizer
-archivedVersions=1.3.2,1.3.3,1.4.0,1.5.0
-publicVersions=2.0.0
+archivedVersions=1.3.2,1.3.3,1.4.0,1.5.0,2.0.0
+publicVersions=2.1.0
 
 defaults.mavenGroupId=com.rigsit
 defaults.mavenArtifactId=sonar-xanitizer-plugin
 
+2.1.0.description=Support for SonarQube 7.9, Added support for JavaScript and TypeScript, Added several new Java problem types
+2.1.0.sqVersions=[7.6,LATEST]
+2.1.0.date=2020-08-10
+2.1.0.changelogUrl=https://github.com/RIGS-IT/sonar-xanitizer/releases/tag/v2.1.0
+2.1.0.downloadUrl=https://github.com/RIGS-IT/sonar-xanitizer/releases/download/v2.1.0/sonar-xanitizer-plugin-2.1.0.jar
+
 2.0.0.description=Support for SonarQube 7.3, Integration into Security Reports, Import All option
-2.0.0.sqVersions=[7.3,LATEST]
+2.0.0.sqVersions=[7.3,7.5]
```

Comment:
This needs to end at the current version, 8.4.1, or users on more recent versions won't be prompted to upgrade